validphys2/src/validphys/core.py

```diff
@@ -545,10 +585,14 @@ def load(self):
         return PositivitySet(cd, fk, self.maxlambda)
 
     def to_unweighted(self):
-        log.warning("Trying to unweight %s, PositivitySetSpec are always unweighted", self.name)
+        log.warning("Trying to unweight %s, %s are always unweighted", self.__class__.__name__, self.name)
         return self
 
 
+class IntegrabilitySetSpec(PositivitySetSpec):
```

Comment:
> Why did you add this? Just to be explicit?

This should've been done long ago.

But we need two of them because they are treated differently at different stages by checking whether the dataset is call "POS" or not...